### ISSUE TITLE
Add support for Identity Service in beta-private-cluster-update-variant

### ISSUE THREAD

**Issue by vasrem** [terraform-google-modules/terraform-google-kubernetes-engine#1137]
### TL;DR

Incorporate the feature introduced by this PR https://github.com/hashicorp/terraform-provider-google-beta/pull/3957 into the [beta-private-cluster-update-variant](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/master/modules/beta-private-cluster-update-variant) in order to be able to enable Identity Service using a flag similarly to [enable_pod_security_policy flag](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/e6b928288cdd79035d32ea84fd8ce6ca40979246/modules/beta-private-cluster-update-variant/variables.tf#L486-L490).

### Terraform Resources

```markdown
* https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster
* https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#nested_identity_service_config
* https://github.com/hashicorp/terraform-provider-google-beta/releases/tag/v4.6.0
```

### Detailed design

```markdown
I believe that this should be implemented similarly to https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/e6b928288cdd79035d32ea84fd8ce6ca40979246/modules/beta-private-cluster-update-variant/cluster.tf#L123-L128. This is a known pattern as far as I can tell from the code so I believe there should not much to be added in the design. Let me know if there are questions and I'll try to answer them.
```
